src/ucs/datastruct/queue.h

```diff
@@ -20,6 +20,7 @@
  */
 static inline void ucs_queue_head_init(ucs_queue_head_t *queue)
 {
+    queue->head  = NULL;
```

Comment:
this can add some overhead for example in tag matching
let's try to avoid this in release mode